src/toast/style.js

```diff
@@ -13,6 +13,122 @@ const Icon = css`
   }
 `;
 
+const fadeOut = keyframes`
+  from {
+    opacity: 1;
+  }
+
+  to {
+    opacity: 0;
+  }
+`;
+
+const slideInTop = keyframes`
+  0% {
+    -webkit-transform: translateY(-500px);
```

Comment:
You don't have to worry about browser prefixes, styled-components adds them automatically.

Reply:
Ah thanks! Was wondering if keyframes left them in or not